integrations/wiz/.port/spec.yaml

```diff
@@ -0,0 +1,23 @@
+type: wiz
+description: Wiz integration for Port Ocean
+icon: Alert
+features:
+  - type: exporter
+    section: Incident Management
+    resources:
+      - kind: issue
+configurations:
+  - name: clientId
+    required: true
+    type: string
+    sensitive: true
+  - name: clientSecret
+    required: true
+    type: string
+    sensitive: true
+  - name: apiUrl
+    type: string
+    required: true
+  - name: tokenUrl
+    type: string
+    required: true
```

Comment:
the convention so far is to prefix these kind of parameters with the integration name

Reply:
alright then.